docs/source/user-guide/tutorials/building-conda-packages.rst

```diff
@@ -422,16 +410,16 @@ section:
     # disable 12 bit jpeg on Windows as we aren't using internal jpeg
     - 0003-disable_jpeg12.patch
 
-This section says that when this package is being built on a Windows
-platform, apply the following patch files. Notice that the patch files
-are in the `patches` directory of the recipe. These patches will only
+This section is basically saying "when this package is being built on a Windows
+platform, apply the following patch files". Notice that the patch files
+are in the ``patches`` directory of the recipe. These patches will only
 be applied to Windows since the ``# [win]`` selector is applied to each
 of the patch entries. For more about selectors, see
 :ref:`preprocess-selectors`.
 
-In the requirements section, notice how there are both a build and
-host set of requirements. For this recipe, all the compilers required to
-build the package are listed in the build requirements.
+In the ``requirements`` section, notice how there are both a ``build`` and
+``host``(??) set of requirements. For this recipe, all the compilers required to
```

Comment:
Are these question marks supposed to be a placeholder for something?

Reply:
Thanks for pointing this out! I thought I'd made changes to this section but apparently I didn't yet! 🙈